Remove Dell Linux laptop option (#250)
meltano/handbook: src/_peopleops/index.md

  As many purchases of laptops have occurred with employees making the purchases and then being reimbursed by Meltano, a remote employee would typically be making a purchase of a laptop pre-loaded with Windows Home Edition.
- Windows Home Edition is notoriously [hard to secure](https://www.markloveless.net/blog/2019/1/15/dealing-with-windows-10).

To standardize the hardware provisioned for Meltano team members, the following Apple hardware option is highly recommended:

> [MacBook Pro - 16” / 512gig SSD / 32gigs RAM / Apple M1 Pro with 10-core CPU, 16-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/16-inch)

Reviewer: ```suggestion
- [14” / 512gig SSD / 32gig RAM / Apple M1 Pro with 10-core CPU, 14-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/14-inch)
- [MacBook Pro - 16” / 512gig SSD / 32gigs RAM / Apple M1 Pro with 10-core CPU, 16-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/16-inch)
```

> Note that the above link is to the Apple store website for the United States. If you are located in a different country, you will likely have to place your order from your regional Apple store website.

Reviewer: ```suggestion
Note that the above links are to the Apple store website for the United States. If you are located in a different country, you will likely have to place your order from your regional Apple store website.
```

We recommend either:
Note that the above link is to the Apple store website for the United States. If you are located in a different country, you will likely have to place your order from your regional Apple store website.

> 1. (MacBook Pro) - [14” / 512gig SSD / 32gig RAM / Apple M1 Pro with 10-core CPU, 14-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/14-inch)

Reviewer 2: @WillDaSilva I think we're okay with people wanting a 14" MacBook.  The 16" one is a bit big for some.

Reviewer: Updated.

  As many purchases of laptops have occurred with employees making the purchases and then being reimbursed by Meltano, a remote employee would typically be making a purchase of a laptop pre-loaded with Windows Home Edition.
- Windows Home Edition is notoriously [hard to secure](https://www.markloveless.net/blog/2019/1/15/dealing-with-windows-10).

To standardize the hardware provisioned for Meltano team members, the following Apple hardware option is highly recommended:

> - [14” / 512gig SSD / 32gig RAM / Apple M1 Pro with 10-core CPU, 14-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/14-inch)
> - [MacBook Pro - 16” / 512gig SSD / 32gigs RAM / Apple M1 Pro with 10-core CPU, 16-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/16-inch)

Reviewer: ```suggestion
- [MacBook Pro 14" / 512gig SSD / 32gig RAM / Apple M1 Pro with 10-core CPU, 14-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/14-inch)
- [MacBook Pro - 16" / 512gig SSD / 32gigs RAM / Apple M1 Pro with 10-core CPU, 16-core GPU, 16-core Neural Engine](https://www.apple.com/shop/buy-mac/macbook-pro/16-inch)
```
